src/styles/components/_footer.scss

```diff
@@ -64,7 +65,13 @@
   @media screen and (prefers-reduced-motion: reduce) {
     &__item > a:hover,
     &__item > a:focus-within {
-      background-color: var(--accent-color);
+      background-color: var(--accent-color-icons);
+    }
+  }
+
+  @media screen and (prefers-color-scheme: dark) {
+    img {
+      filter: invert(100);
```

Comment:
Does this need a percent sign for 100%? Or should we use a number between 0 and 1 like the filters a little further down in the file? Like lines 110 or 117? I don't really have a preference either way, but we should do it all the same.

Reply:
I think w/ invert you need to use a value 0-100 but I will double check to be sure!

Reply:
I changed it to 1 per the [MDN guidelines](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert). It goes from 0-1 and values in between are % (like 25 would be written 25%). Thanks for having me check that!

![image](https://user-images.githubusercontent.com/49820744/200976398-a6c9ff33-36a3-4706-a23f-977e0674f4d8.png)